phoenix-shared/src/commonMain/kotlin/fr.acinq.phoenix/managers/NodeParamsManager.kt

```diff
@@ -66,14 +66,24 @@ class NodeParamsManager(
                     keyManager = keyManager,
                     alias = "phoenix",
                     features = Features(
```

Comment:
You also need to add `Feature.ExperimentalTrampolinePayment to FeatureSupport.Optional` and we should be good to go! Then can you generate an invoice and send it to me, I'll verify that everything looks good?

Reply:
Testnet : 
```
lntb1p3x4rndpp589qu76gypsvpv50fvf45jm68fuhm99q6s6dnzl7swsyhqmapwnzqdqqcqpjsp5ggt5f6gazu05d8up2zhhqnzjz3yjrc8c8fw7yd2hn0f5d955ttsq9q7sqqqqqqqqqqqqqqqqqqzsqqqqqysgqmqz9gxqyjw5qrzjqwfn3p9278ttzzpe0e00uhyxhned3j5d9acqak5emwfpflp8z2cnfla87pchnswvmyqqqqlgqqqqqeqqjqjxqluc5znccs4nauufv7te9daq7e9eeaejd6p5c9l660lw3t5k3h02qys6cfaryl93052gl8y0uys8mgkeqqymcuc2axdl6j95r9zqcpkkuuc9
```

Mainnet :
```
lnbc1p3x4r7ypp52080vpag6nf8vpt2uyvymlchnhjlyx2ag3ddq6uqskwfad5vtm0sdqqcqpjsp53pp7jjgynw54v5yzexd0zqdwppz8rx76zctpn4rflcvm36kms60q9q7sqqqqqqqqqqqqqqqqqqzsqqqqqysgqmqz9gxqyjw5qrzjqwryaup9lh50kkranzgcdnn2fgvx390wgj5jd07rwr3vxeje0glcllu7kzl470wpz5qqqqlgqqqqqeqqjqsndqyhj0gzvsy76h0srtgw0u0kwz4u3tx9nkffy37t3j56pd2lvj5547j7kv3jv9hac7zrafap0j8pp6ywvy03cj4wyd7ltc3wpu2msqgp4xlh
```